### ISSUE TITLE
Playout doesn't work (on air not lit) - Also playback of files generates "Oops! Something went wrong!"

### ISSUE THREAD

**Issue by itcrowdsource** [LibreTime/libretime#567]
My LibreTime instance stopped working. I can't get it to broadcast anymore (on air not lit)
![onair](https://user-images.githubusercontent.com/20809792/47149689-9769e780-d2d4-11e8-8f92-1f935c9ed351.JPG)

Also playing back files is generating an "Oops! Something went wrong!" in the player pop-up window. 
![track playback](https://user-images.githubusercontent.com/20809792/47149612-67224900-d2d4-11e8-9eac-6ad5b861da2f.JPG)

pypo.log
```
2018-10-18 10:27:00,421 [pypofetch] [INFO ]  Queue timeout. Fetching schedule manually
2018-10-18 10:27:00,801 [pypofetch] [INFO ]  Loop #1482
2018-10-18 10:27:00,801 [pypofetch] [INFO ]  Queue timeout. Fetching schedule manually
2018-10-18 10:27:01,297 [pypofetch] [INFO ]  Loop #1483
2018-10-18 10:27:01,298 [pypofetch] [INFO ]  Queue timeout. Fetching schedule manually
2018-10-18 10:27:01,659 [pypofetch] [INFO ]  Loop #1484
2018-10-18 10:27:01,659 [pypofetch] [INFO ]  Queue timeout. Fetching schedule manually
2018-10-18 10:27:02,031 [pypofetch] [INFO ]  Loop #1485
2018-10-18 10:27:02,031 [pypofetch] [INFO ]  Queue timeout. Fetching schedule manually
```

airtime_analyzer.log
```
2018-10-18 09:22:25,490 [message_listener] [ERROR]  Connection to message queue failed.
2018-10-18 09:22:25,497 [message_listener] [ERROR]
2018-10-18 09:22:25,497 [message_listener] [INFO ]  Retrying in 5 seconds...
2018-10-18 09:22:30,503 [message_listener] [ERROR]  Connection to message queue failed.
2018-10-18 09:22:30,504 [message_listener] [ERROR]
2018-10-18 09:22:30,504 [message_listener] [INFO ]  Retrying in 5 seconds...
2018-10-18 09:22:35,506 [message_listener] [ERROR]  Connection to message queue failed.
2018-10-18 09:22:35,507 [message_listener] [ERROR]
2018-10-18 09:22:35,507 [message_listener] [INFO ]  Retrying in 5 seconds...
2018-10-18 09:22:40,524 [message_listener] [INFO ]   Listening for messages...
```

notify.log
```
2018-10-18 10:24:01,792 [pyponotify] [INFO ]  # Calling server to update liquidsoap status    #
2018-10-18 10:24:01,793 [pyponotify] [INFO ]  #################################################
2018-10-18 10:24:01,795 [pyponotify] [INFO ]  msg = OK
2018-10-18 10:24:01,829 [pyponotify] [INFO ]  Response: null
2018-10-18 10:24:01,897 [pyponotify] [INFO ]  #################################################
2018-10-18 10:24:01,910 [pyponotify] [INFO ]  # Calling server to update liquidsoap status    #
2018-10-18 10:24:01,910 [pyponotify] [INFO ]  #################################################
2018-10-18 10:24:01,914 [pyponotify] [INFO ]  msg = OK
2018-10-18 10:24:01,982 [pyponotify] [INFO ]  Response: null
2018-10-18 10:24:02,063 [pyponotify] [INFO ]  Response: null
```

zendphp.log
```
2018-10-18T10:37:31+00:00 ERR (3): <servername> [ErrorController.php:54 - errorAction()] - An internal application error has occurred.: PDOException: SQLSTATE[42P01]: Undefined table: 7 ERROR:  relation "cloud_file" does not exist
LINE 1: ...loud_file.resource_id, cloud_file.cc_file_id FROM cloud_file...
                                                             ^ in /usr/share/airtime/php/vendor/propel/propel1/runtime/lib/query/ModelCriteria.php:1386
Stack trace:
#0 /usr/share/airtime/php/vendor/propel/propel1/runtime/lib/query/ModelCriteria.php(1386): PDOStatement->execute()
#1 /usr/share/airtime/php/vendor/propel/propel1/runtime/lib/query/ModelCriteria.php(1249): ModelCriteria->doSelect(Object(PropelPDO))
#2 /usr/share/airtime/php/vendor/propel/propel1/runtime/lib/query/ModelCriteria.php(1456): ModelCriteria->findOne(Object(PropelPDO))
#3 /usr/share/airtime/php/vendor/propel/propel1/runtime/lib/query/ModelCriteria.php(2255): ModelCriteria->findOneBy('CcFileId', 12040)
#4 /usr/share/airtime/php/airtime_mvc/application/models/StoredFile.php(607): ModelCriteria->__call('findOneByCcFile...', Array)
#5 /usr/share/airtime/php/airtime_mvc/application/models/Schedule.php(1029): Application_Model_StoredFile::RecallById(12040)
#6 /usr/share/airtime/php/airtime_mvc/application/models/Schedule.php(1133): Application_Model_Schedule::createScheduledEvents(Array, '2018-10-18 10:3...', '2018-10-18 11:3...')
#7 /usr/share/airtime/php/airtime_mvc/application/controllers/ApiController.php(566): Application_Model_Schedule::getSchedule()
#8 /usr/share/airtime/php/vendor/zendframework/zendframework1/library/Zend/Controller/Action.php(516): ApiController->scheduleAction()
#9 /usr/share/airtime/php/vendor/zendframework/zendframework1/library/Zend/Controller/Dispatcher/Standard.php(308): Zend_Controller_Action->dispatch('scheduleAction')
#10 /usr/share/airtime/php/vendor/zendframework/zendframework1/library/Zend/Controller/Front.php(954): Zend_Controller_Dispatcher_Standard->dispatch(Object(Zend_Controller_Request_Http), Object(Zend_Controller_Response_Http))
#11 /usr/share/airtime/php/vendor/zendframework/zendframework1/library/Zend/Application/Bootstrap/Bootstrap.php(105): Zend_Controller_Front->dispatch()
#12 /usr/share/airtime/php/vendor/zendframework/zendframework1/library/Zend/Application.php(384): Zend_Application_Bootstrap_Bootstrap->run()
#13 /usr/share/airtime/php/airtime_mvc/application/airtime-boot.php(84): Zend_Application->run()
#14 /usr/share/airtime/php/airtime_mvc/public/index.php(68): require_once('/usr/share/airt...')
#15 {main}

Next PropelException: Unable to execute SELECT statement [SELECT cloud_file.id, cloud_file.storage_backend, cloud_file.resource_id, cloud_file.cc_file_id FROM cloud_file WHERE cloud_file.cc_file_id=:p1 LIMIT 1] [wrapped: SQLSTATE[42P01]: Undefined table: 7 ERROR:  relation "cloud_file" does not exist
LINE 1: ...loud_file.resource_id, cloud_file.cc_file_id FROM cloud_file...
                                                             ^] in /usr/share/airtime/php/vendor/propel/propel1/runtime/lib/query/ModelCriteria.php:1389
Stack trace:
#0 /usr/share/airtime/php/vendor/propel/propel1/runtime/lib/query/ModelCriteria.php(1249): ModelCriteria->doSelect(Object(PropelPDO))
#1 /usr/share/airtime/php/vendor/propel/propel1/runtime/lib/query/ModelCriteria.php(1456): ModelCriteria->findOne(Object(PropelPDO))
#2 /usr/share/airtime/php/vendor/propel/propel1/runtime/lib/query/ModelCriteria.php(2255): ModelCriteria->findOneBy('CcFileId', 12040)
#3 /usr/share/airtime/php/airtime_mvc/application/models/StoredFile.php(607): ModelCriteria->__call('findOneByCcFile...', Array)
#4 /usr/share/airtime/php/airtime_mvc/application/models/Schedule.php(1029): Application_Model_StoredFile::RecallById(12040)
#5 /usr/share/airtime/php/airtime_mvc/application/models/Schedule.php(1133): Application_Model_Schedule::createScheduledEvents(Array, '2018-10-18 10:3...', '2018-10-18 11:3...')
#6 /usr/share/airtime/php/airtime_mvc/application/controllers/ApiController.php(566): Application_Model_Schedule::getSchedule()
#7 /usr/share/airtime/php/vendor/zendframework/zendframework1/library/Zend/Controller/Action.php(516): ApiController->scheduleAction()
#8 /usr/share/airtime/php/vendor/zendframework/zendframework1/library/Zend/Controller/Dispatcher/Standard.php(308): Zend_Controller_Action->dispatch('scheduleAction')
#9 /usr/share/airtime/php/vendor/zendframework/zendframework1/library/Zend/Controller/Front.php(954): Zend_Controller_Dispatcher_Standard->dispatch(Object(Zend_Controller_Request_Http), Object(Zend_Controller_Response_Http))
#10 /usr/share/airtime/php/vendor/zendframework/zendframework1/library/Zend/Application/Bootstrap/Bootstrap.php(105): Zend_Controller_Front->dispatch()
#11 /usr/share/airtime/php/vendor/zendframework/zendframework1/library/Zend/Application.php(384): Zend_Application_Bootstrap_Bootstrap->run()
#12 /usr/share/airtime/php/airtime_mvc/application/airtime-boot.php(84): Zend_Application->run()
#13 /usr/share/airtime/php/airtime_mvc/public/index.php(68): require_once('/usr/share/airt...')
#14 {main}
2018-10-18T10:37:32+00:00 ERR (3): <servername> [ErrorController.php:26 - errorAction()] - Unable to execute SELECT statement [SELECT cloud_file.id, cloud_file.storage_backend, cloud_file.resource_id, cloud_file.cc_file_id FROM cloud_file WHERE cloud_file.cc_file_id=:p1 LIMIT 1] [wrapped: SQLSTATE[42P01]: Undefined table: 7 ERROR:  relation "cloud_file" does not exist
LINE 1: ...loud_file.resource_id, cloud_file.cc_file_id FROM cloud_file...
                                                             ^]
2018-10-18T10:37:32+00:00 ERR (3): live.fnoobtechno.com [ErrorController.php:27 - errorAction()] - #0 /usr/share/airtime/php/vendor/propel/propel1/runtime/lib/query/ModelCriteria.php(1249): ModelCriteria->doSelect(Object(PropelPDO))
#1 /usr/share/airtime/php/vendor/propel/propel1/runtime/lib/query/ModelCriteria.php(1456): ModelCriteria->findOne(Object(PropelPDO))
#2 /usr/share/airtime/php/vendor/propel/propel1/runtime/lib/query/ModelCriteria.php(2255): ModelCriteria->findOneBy('CcFileId', 12040)
#3 /usr/share/airtime/php/airtime_mvc/application/models/StoredFile.php(607): ModelCriteria->__call('findOneByCcFile...', Array)
#4 /usr/share/airtime/php/airtime_mvc/application/models/Schedule.php(1029): Application_Model_StoredFile::RecallById(12040)
#5 /usr/share/airtime/php/airtime_mvc/application/models/Schedule.php(1133): Application_Model_Schedule::createScheduledEvents(Array, '2018-10-18 10:3...', '2018-10-18 11:3...')
#6 /usr/share/airtime/php/airtime_mvc/application/controllers/ApiController.php(566): Application_Model_Schedule::getSchedule()
#7 /usr/share/airtime/php/vendor/zendframework/zendframework1/library/Zend/Controller/Action.php(516): ApiController->scheduleAction()
#8 /usr/share/airtime/php/vendor/zendframework/zendframework1/library/Zend/Controller/Dispatcher/Standard.php(308): Zend_Controller_Action->dispatch('scheduleAction')
#9 /usr/share/airtime/php/vendor/zendframework/zendframework1/library/Zend/Controller/Front.php(954): Zend_Controller_Dispatcher_Standard->dispatch(Object(Zend_Controller_Request_Http), Object(Zend_Controller_Response_Http))
#10 /usr/share/airtime/php/vendor/zendframework/zendframework1/library/Zend/Application/Bootstrap/Bootstrap.php(105): Zend_Controller_Front->dispatch()
#11 /usr/share/airtime/php/vendor/zendframework/zendframework1/library/Zend/Application.php(384): Zend_Application_Bootstrap_Bootstrap->run()
#12 /usr/share/airtime/php/airtime_mvc/application/airtime-boot.php(84): Zend_Application->run()
#13 /usr/share/airtime/php/airtime_mvc/public/index.php(68): require_once('/usr/share/airt...')
#14 {main}
2018-10-18T10:37:32+00:00 ERR (3): live.fnoobtechno.com [ErrorController.php:54 - errorAction()] - An internal application error has occurred.: PDOException: SQLSTATE[42P01]: Undefined table: 7 ERROR:  relation "cloud_file" does not exist
LINE 1: ...loud_file.resource_id, cloud_file.cc_file_id FROM cloud_file...
                                                             ^ in /usr/share/airtime/php/vendor/propel/propel1/runtime/lib/query/ModelCriteria.php:1386
Stack trace:
#0 /usr/share/airtime/php/vendor/propel/propel1/runtime/lib/query/ModelCriteria.php(1386): PDOStatement->execute()
#1 /usr/share/airtime/php/vendor/propel/propel1/runtime/lib/query/ModelCriteria.php(1249): ModelCriteria->doSelect(Object(PropelPDO))
#2 /usr/share/airtime/php/vendor/propel/propel1/runtime/lib/query/ModelCriteria.php(1456): ModelCriteria->findOne(Object(PropelPDO))
#3 /usr/share/airtime/php/vendor/propel/propel1/runtime/lib/query/ModelCriteria.php(2255): ModelCriteria->findOneBy('CcFileId', 12040)
#4 /usr/share/airtime/php/airtime_mvc/application/models/StoredFile.php(607): ModelCriteria->__call('findOneByCcFile...', Array)
#5 /usr/share/airtime/php/airtime_mvc/application/models/Schedule.php(1029): Application_Model_StoredFile::RecallById(12040)
#6 /usr/share/airtime/php/airtime_mvc/application/models/Schedule.php(1133): Application_Model_Schedule::createScheduledEvents(Array, '2018-10-18 10:3...', '2018-10-18 11:3...')
#7 /usr/share/airtime/php/airtime_mvc/application/controllers/ApiController.php(566): Application_Model_Schedule::getSchedule()
#8 /usr/share/airtime/php/vendor/zendframework/zendframework1/library/Zend/Controller/Action.php(516): ApiController->scheduleAction()
#9 /usr/share/airtime/php/vendor/zendframework/zendframework1/library/Zend/Controller/Dispatcher/Standard.php(308): Zend_Controller_Action->dispatch('scheduleAction')
#10 /usr/share/airtime/php/vendor/zendframework/zendframework1/library/Zend/Controller/Front.php(954): Zend_Controller_Dispatcher_Standard->dispatch(Object(Zend_Controller_Request_Http), Object(Zend_Controller_Response_Http))
#11 /usr/share/airtime/php/vendor/zendframework/zendframework1/library/Zend/Application/Bootstrap/Bootstrap.php(105): Zend_Controller_Front->dispatch()
#12 /usr/share/airtime/php/vendor/zendframework/zendframework1/library/Zend/Application.php(384): Zend_Application_Bootstrap_Bootstrap->run()
#13 /usr/share/airtime/php/airtime_mvc/application/airtime-boot.php(84): Zend_Application->run()
#14 /usr/share/airtime/php/airtime_mvc/public/index.php(68): require_once('/usr/share/airt...')
#15 {main}

Next PropelException: Unable to execute SELECT statement [SELECT cloud_file.id, cloud_file.storage_backend, cloud_file.resource_id, cloud_file.cc_file_id FROM cloud_file WHERE cloud_file.cc_file_id=:p1 LIMIT 1] [wrapped: SQLSTATE[42P01]: Undefined table: 7 ERROR:  relation "cloud_file" does not exist
LINE 1: ...loud_file.resource_id, cloud_file.cc_file_id FROM cloud_file...
                                                             ^] in /usr/share/airtime/php/vendor/propel/propel1/runtime/lib/query/ModelCriteria.php:1389
Stack trace:
#0 /usr/share/airtime/php/vendor/propel/propel1/runtime/lib/query/ModelCriteria.php(1249): ModelCriteria->doSelect(Object(PropelPDO))
#1 /usr/share/airtime/php/vendor/propel/propel1/runtime/lib/query/ModelCriteria.php(1456): ModelCriteria->findOne(Object(PropelPDO))
#2 /usr/share/airtime/php/vendor/propel/propel1/runtime/lib/query/ModelCriteria.php(2255): ModelCriteria->findOneBy('CcFileId', 12040)
#3 /usr/share/airtime/php/airtime_mvc/application/models/StoredFile.php(607): ModelCriteria->__call('findOneByCcFile...', Array)
#4 /usr/share/airtime/php/airtime_mvc/application/models/Schedule.php(1029): Application_Model_StoredFile::RecallById(12040)
#5 /usr/share/airtime/php/airtime_mvc/application/models/Schedule.php(1133): Application_Model_Schedule::createScheduledEvents(Array, '2018-10-18 10:3...', '2018-10-18 11:3...')
#6 /usr/share/airtime/php/airtime_mvc/application/controllers/ApiController.php(566): Application_Model_Schedule::getSchedule()
#7 /usr/share/airtime/php/vendor/zendframework/zendframework1/library/Zend/Controller/Action.php(516): ApiController->scheduleAction()
#8 /usr/share/airtime/php/vendor/zendframework/zendframework1/library/Zend/Controller/Dispatcher/Standard.php(308): Zend_Controller_Action->dispatch('scheduleAction')
#9 /usr/share/airtime/php/vendor/zendframework/zendframework1/library/Zend/Controller/Front.php(954): Zend_Controller_Dispatcher_Standard->dispatch(Object(Zend_Controller_Request_Http), Object(Zend_Controller_Response_Http))
#10 /usr/share/airtime/php/vendor/zendframework/zendframework1/library/Zend/Application/Bootstrap/Bootstrap.php(105): Zend_Controller_Front->dispatch()
#11 /usr/share/airtime/php/vendor/zendframework/zendframework1/library/Zend/Application.php(384): Zend_Application_Bootstrap_Bootstrap->run()
#12 /usr/share/airtime/php/airtime_mvc/application/airtime-boot.php(84): Zend_Application->run()
#13 /usr/share/airtime/php/airtime_mvc/public/index.php(68): require_once('/usr/share/airt...')
#14 {main}
2018-10-18T10:37:32+00:00 ERR (3): live.fnoobtechno.com [ErrorController.php:26 - errorAction()] - Unable to execute SELECT statement [SELECT cloud_file.id, cloud_file.storage_backend, cloud_file.resource_id, cloud_file.cc_file_id FROM cloud_file WHERE cloud_file.cc_file_id=:p1 LIMIT 1] [wrapped: SQLSTATE[42P01]: Undefined table: 7 ERROR:  relation "cloud_file" does not exist
LINE 1: ...loud_file.resource_id, cloud_file.cc_file_id FROM cloud_file...
                                                             ^]
2018-10-18T10:37:32+00:00 ERR (3): live.fnoobtechno.com [ErrorController.php:27 - errorAction()] - #0 /usr/share/airtime/php/vendor/propel/propel1/runtime/lib/query/ModelCriteria.php(1249): ModelCriteria->doSelect(Object(PropelPDO))
#1 /usr/share/airtime/php/vendor/propel/propel1/runtime/lib/query/ModelCriteria.php(1456): ModelCriteria->findOne(Object(PropelPDO))
#2 /usr/share/airtime/php/vendor/propel/propel1/runtime/lib/query/ModelCriteria.php(2255): ModelCriteria->findOneBy('CcFileId', 12040)
#3 /usr/share/airtime/php/airtime_mvc/application/models/StoredFile.php(607): ModelCriteria->__call('findOneByCcFile...', Array)
#4 /usr/share/airtime/php/airtime_mvc/application/models/Schedule.php(1029): Application_Model_StoredFile::RecallById(12040)
#5 /usr/share/airtime/php/airtime_mvc/application/models/Schedule.php(1133): Application_Model_Schedule::createScheduledEvents(Array, '2018-10-18 10:3...', '2018-10-18 11:3...')
#6 /usr/share/airtime/php/airtime_mvc/application/controllers/ApiController.php(566): Application_Model_Schedule::getSchedule()
#7 /usr/share/airtime/php/vendor/zendframework/zendframework1/library/Zend/Controller/Action.php(516): ApiController->scheduleAction()
#8 /usr/share/airtime/php/vendor/zendframework/zendframework1/library/Zend/Controller/Dispatcher/Standard.php(308): Zend_Controller_Action->dispatch('scheduleAction')
#9 /usr/share/airtime/php/vendor/zendframework/zendframework1/library/Zend/Controller/Front.php(954): Zend_Controller_Dispatcher_Standard->dispatch(Object(Zend_Controller_Request_Http), Object(Zend_Controller_Response_Http))
#10 /usr/share/airtime/php/vendor/zendframework/zendframework1/library/Zend/Application/Bootstrap/Bootstrap.php(105): Zend_Controller_Front->dispatch()
#11 /usr/share/airtime/php/vendor/zendframework/zendframework1/library/Zend/Application.php(384): Zend_Application_Bootstrap_Bootstrap->run()
#12 /usr/share/airtime/php/airtime_mvc/application/airtime-boot.php(84): Zend_Application->run()
#13 /usr/share/airtime/php/airtime_mvc/public/index.php(68): require_once('/usr/share/airt...')
#14 {main}
2018-10-18T10:37:32+00:00 ERR (3): <servername> [ErrorController.php:54 - errorAction()] - An internal application error has occurred.: PDOException: SQLSTATE[42P01]: Undefined table: 7 ERROR:  relation "cloud_file" does not exist
LINE 1: ...loud_file.resource_id, cloud_file.cc_file_id FROM cloud_file...
                                                             ^ in /usr/share/airtime/php/vendor/propel/propel1/runtime/lib/query/ModelCriteria.php:1386
Stack trace:
#0 /usr/share/airtime/php/vendor/propel/propel1/runtime/lib/query/ModelCriteria.php(1386): PDOStatement->execute()
#1 /usr/share/airtime/php/vendor/propel/propel1/runtime/lib/query/ModelCriteria.php(1249): ModelCriteria->doSelect(Object(PropelPDO))
#2 /usr/share/airtime/php/vendor/propel/propel1/runtime/lib/query/ModelCriteria.php(1456): ModelCriteria->findOne(Object(PropelPDO))
#3 /usr/share/airtime/php/vendor/propel/propel1/runtime/lib/query/ModelCriteria.php(2255): ModelCriteria->findOneBy('CcFileId', 12040)
#4 /usr/share/airtime/php/airtime_mvc/application/models/StoredFile.php(607): ModelCriteria->__call('findOneByCcFile...', Array)
#5 /usr/share/airtime/php/airtime_mvc/application/models/Schedule.php(1029): Application_Model_StoredFile::RecallById(12040)
#6 /usr/share/airtime/php/airtime_mvc/application/models/Schedule.php(1133): Application_Model_Schedule::createScheduledEvents(Array, '2018-10-18 10:3...', '2018-10-18 11:3...')
#7 /usr/share/airtime/php/airtime_mvc/application/controllers/ApiController.php(566): Application_Model_Schedule::getSchedule()
#8 /usr/share/airtime/php/vendor/zendframework/zendframework1/library/Zend/Controller/Action.php(516): ApiController->scheduleAction()
#9 /usr/share/airtime/php/vendor/zendframework/zendframework1/library/Zend/Controller/Dispatcher/Standard.php(308): Zend_Controller_Action->dispatch('scheduleAction')
#10 /usr/share/airtime/php/vendor/zendframework/zendframework1/library/Zend/Controller/Front.php(954): Zend_Controller_Dispatcher_Standard->dispatch(Object(Zend_Controller_Request_Http), Object(Zend_Controller_Response_Http))
#11 /usr/share/airtime/php/vendor/zendframework/zendframework1/library/Zend/Application/Bootstrap/Bootstrap.php(105): Zend_Controller_Front->dispatch()
#12 /usr/share/airtime/php/vendor/zendframework/zendframework1/library/Zend/Application.php(384): Zend_Application_Bootstrap_Bootstrap->run()
#13 /usr/share/airtime/php/airtime_mvc/application/airtime-boot.php(84): Zend_Application->run()
#14 /usr/share/airtime/php/airtime_mvc/public/index.php(68): require_once('/usr/share/airt...')
#15 {main}

Next PropelException: Unable to execute SELECT statement [SELECT cloud_file.id, cloud_file.storage_backend, cloud_file.resource_id, cloud_file.cc_file_id FROM cloud_file WHERE cloud_file.cc_file_id=:p1 LIMIT 1] [wrapped: SQLSTATE[42P01]: Undefined table: 7 ERROR:  relation "cloud_file" does not exist
LINE 1: ...loud_file.resource_id, cloud_file.cc_file_id FROM cloud_file...
                                                             ^] in /usr/share/airtime/php/vendor/propel/propel1/runtime/lib/query/ModelCriteria.php:1389
Stack trace:
#0 /usr/share/airtime/php/vendor/propel/propel1/runtime/lib/query/ModelCriteria.php(1249): ModelCriteria->doSelect(Object(PropelPDO))
#1 /usr/share/airtime/php/vendor/propel/propel1/runtime/lib/query/ModelCriteria.php(1456): ModelCriteria->findOne(Object(PropelPDO))
#2 /usr/share/airtime/php/vendor/propel/propel1/runtime/lib/query/ModelCriteria.php(2255): ModelCriteria->findOneBy('CcFileId', 12040)
#3 /usr/share/airtime/php/airtime_mvc/application/models/StoredFile.php(607): ModelCriteria->__call('findOneByCcFile...', Array)
#4 /usr/share/airtime/php/airtime_mvc/application/models/Schedule.php(1029): Application_Model_StoredFile::RecallById(12040)
#5 /usr/share/airtime/php/airtime_mvc/application/models/Schedule.php(1133): Application_Model_Schedule::createScheduledEvents(Array, '2018-10-18 10:3...', '2018-10-18 11:3...')
#6 /usr/share/airtime/php/airtime_mvc/application/controllers/ApiController.php(566): Application_Model_Schedule::getSchedule()
#7 /usr/share/airtime/php/vendor/zendframework/zendframework1/library/Zend/Controller/Action.php(516): ApiController->scheduleAction()
#8 /usr/share/airtime/php/vendor/zendframework/zendframework1/library/Zend/Controller/Dispatcher/Standard.php(308): Zend_Controller_Action->dispatch('scheduleAction')
#9 /usr/share/airtime/php/vendor/zendframework/zendframework1/library/Zend/Controller/Front.php(954): Zend_Controller_Dispatcher_Standard->dispatch(Object(Zend_Controller_Request_Http), Object(Zend_Controller_Response_Http))
#10 /usr/share/airtime/php/vendor/zendframework/zendframework1/library/Zend/Application/Bootstrap/Bootstrap.php(105): Zend_Controller_Front->dispatch()
#11 /usr/share/airtime/php/vendor/zendframework/zendframework1/library/Zend/Application.php(384): Zend_Application_Bootstrap_Bootstrap->run()
#12 /usr/share/airtime/php/airtime_mvc/application/airtime-boot.php(84): Zend_Application->run()
#13 /usr/share/airtime/php/airtime_mvc/public/index.php(68): require_once('/usr/share/airt...')
```

**Comment by Robbt** [comment]
Ok, this is concerning. I don't know what caused this error to occur. We recently removed some of the config related to cloud support, so it may just be that we need to remove this code that attempts to query a cloud file object. I don't think we removed the object from the database though, so I don't know why this cropped up. 

Was it working before ? If so do you know what you did right before it quit working ? If you commented out the lines related to the cloud check in StoreFile.php it would be a quick fix that might work. Without figuring out why it happened I don't want to just start throwing in PRs.

**Comment by itcrowdsource** [comment]
Hi @Robbt :-)

The system has worked properly before, I've been testing it for a few months. 

The things I did:
- Installed an SSL certificate to support https.
- An import of my current Airtime database and some mp3 audio files. I've done the import many times without running into problems. 
- Installed the latest LibreTime version (probably the culprit?)

I found the file you mentioned, I believe it's StoredFile.php and not StoreFile.php right?
Could you point me the section/lines to comment out? That would be very helpful;-)

**Comment by hairmare** [comment]
I just stared at the changes in the last release for a bit. It doesn't look like we changed anything that would explain the `Undefined table: 7 ERROR:  relation "cloud_file" does not exist` error you are seeing.

Maybe the issue is somewhere in the following lines:

https://github.com/LibreTime/libretime/blob/5b822afe14046157a8d9fbdff103eb383673e6be/airtime_mvc/application/models/StoredFile.php#L604-L613

You might want to try commenting line 607 by prefixing it with `//` and replacing line 609 with `if (true) {`.

**Comment by Robbt** [comment]
So yeah, I think what Lucas suggests would fix it.
My only guess is that somehow when you were pulling your Airtime database it didn't get the new fields added by LibreTime created (not sure why it happened this time but not others).

If that is the case it might mean the upgrade SQL scripts didn't run this time but did run other times. I haven't looked specifically to see if this field was added in one of the upgrades. So you should verify that you have the latest database or else things like auto playlist might not work and podcasts etc.

If everything else looks fine then this workaround should work as we don't do anything with cloudfile stuff.

**Comment by itcrowdsource** [comment]
@Robbt @hairmare I've applied the changes and the issue's gone. So the cloud_file was the culprit.

**Comment by hairmare** [comment]
I'll give reproducing this a try over the weekend, if it's really an issue with the latest alpha I'll probably end up pushing a new release with a fix soon.

**Comment by hairmare** [comment]
I can't reproduce this, but I don't seem to know enough about your db state. The most likely explanation is that the following migration was not applied for some reason.

https://github.com/LibreTime/libretime/blob/5b822afe14046157a8d9fbdff103eb383673e6be/airtime_mvc/application/controllers/upgrade_sql/airtime_2.5.9/upgrade.sql

What version of Airtime is your db, dump from? Was it a fresh import into LibreTime or a migration from a previous version? And does the the `cloud_file` table really not exist in the db dump?

**Comment by itcrowdsource** [comment]
@hairmare 
That makes sense. This is what I did to migrate from Airtime to LibreTime:

1. Created a fresh install of LibreTime
2. Made a database export of Airtime (2.5.1)
3. Dropped the Libretime database
4. Create a new database on the LibreTime server
5. Imported the Airtime database on the LibreTime server
6. Ran the LibreTime install again

So basically I'm missing these database upgrade scripts then?
https://github.com/LibreTime/libretime/tree/5b822afe14046157a8d9fbdff103eb383673e6be/airtime_mvc/application/controllers/upgrade_sql

Or is there another procedure to properly migrate the content from an existing Airtime database into LibreTime?

**Comment by Robbt** [comment]
Basically you can just install LibreTime on top of the Airtime database. I'd back up the airtime database or duplicate it and then point LibreTime at the copied database. So yeah by importing it onto an already installed LibreTime setup you missed the upgrade scripts. So I'm going to close this as the issue was the way that you did the upgrade and not a problem with the code per se.

**Comment by hairmare** [comment]
I think in the long run we would want to support updating by just pointing LibreTime at an Airtime database and being done. I'm not quite sure why it failed.

> 1. Created a fresh install of LibreTime

Did you go through the web based setup in the installation?

> 2. Made a database export of Airtime (2.5.1)

This is most likely the culprit, I believe you are missing out on the 2.5.2 db changes and LibreTime assumes that everything starts with an up to date 2.5.2 install, I'm not 100% sure on this but if i recall correctly there where some changes to the migration scripting in upstream at that point.

> 3. Dropped the Libretime database
> 4. Create a new database on the LibreTime server
> 5. Imported the Airtime database on the LibreTime server

This is exactly how it "should" work!

> 6. Ran the LibreTime install again

I don't think there is a real need for this step. Migrations get run on the first request to the system.

When I did the switch I migrated the config and the database in one go, so you might want to try also adding your old airtime.conf to the migration.
